_data/utilities.yml

```diff
@@ -150,6 +150,14 @@ websites:
       img: optus.png
       tfa: No
 
+    - name: Orange
+      url: https://orange.fr/
```

Comment:
The current `url` redirects to a HTTP page for me. @divayprakash, please use `https://www.orange.fr/portail` instead.